vivo-project/Vitro
# Replaced PlainGraph with GraphMem as a workaround for Jena plain grap…

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues/3977)**

# What does this pull request do?
Fixes issue related to removing statements with language strings from configuration graphs. 

# What's new?
Replaced Graph implementation used in BulkUpdate models

# How should this be tested?
* Reproduce the problem in the issue, check that PR fixes the issue
* Run tests provided with the PR

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers

# Reviewers' expertise
Candidates for reviewing this PR should have some of the following expertises:
Java, Jena

# Reviewers' report template

## General comment
A reviewer should provide here comments and suggestions for requested changes if any.
## Testing
A reviewer should briefly describe here how it was tested
## Code reviewing
A reviewer should briefly describe here which part was code reviewed
